### ISSUE TITLE
Need Nuxt - tailwind example with custom config

### ISSUE THREAD

**Issue by ramsane** [cypress-io/cypress#9686]
I have a project setup with Nuxt typescript and tailwind custom configuration. I was able to setup cypress component testing with the instructions in the link. https://github.com/bahmutov/cypress-vue-unit-test/blob/master/docs/manual-install.md

But it gave me error in the cypress window.
```
The function exported by the plugins file threw an error.
We invoked the function exported by C:\workspace\QRPlatform\tests\e2e\plugins\index.js, but it threw an error.
Cannot read property 'rules' of undefined.
```
Then I was able to setup this with cypress-nuxt. But when I launch a simple component, none of the vue logic(some click button) and tailwind config(custom config) seems to be working.

Is there any working guide with few examples of testing tailwind components ?

**Comment by travis5491811** [comment]
@ramsane can you post what you have for setup? The url you posted is 404 now. There are docs for manual install at [a different url](https://github.com/cypress-io/cypress/blob/master/npm/vue/docs/manual-install.md) than the one you posted. The instructions i found do not work for `nuxt` but i've troubleshooted enough to get everything but weback reading `.vue` files. I'm trying to figure out the last part so I can do a pull request that should resolve this and https://github.com/cypress-io/cypress/issues/9721

Basically my changes so far are adding: `npm install --save-dev @cypress/webpack-dev-server webpack-dev-server` to resolve errors that would not let cypress start. I also looked at what `vue add cypress-experimental` did in a regular vue-cli project and didn't see anything extra in the support or plugins setup.

**Comment by ramsane** [comment]
@travis5491811 Thank you for the response. But, I followed the different url that was mentioned there, but I left it long time ago. Now, this is not part of my project. I would like to try it some other time. But if there is a working example with nuxt and tailwind, it would be great. I tried few things back then. But as that was not a stable one, I left that.

We can close this one here.
